tests/test_jax_nonbonded.py

```diff
@@ -34,12 +29,19 @@
     nonbonded_on_specific_pairs,
 )
 
-Conf = Params = Box = ChargeMask = LJMask = LambdaPlaneIdxs = LambdaOffsetIdxs = jnp.array
-Lamb = Beta = Cutoff = Energy = float
+Array = Any
+Conf = Array
+Params = Array
+Box = Array
+ChargeMask = Array
+LJMask = Array
+Lamb = float
+Beta = float
+Cutoff = float
+Energy = float
 
-nonbonded_args = Conf, Params, Box, Lamb, ChargeMask, LJMask, Beta, Cutoff, LambdaPlaneIdxs, LambdaOffsetIdxs
-NonbondedArgs = Tuple[nonbonded_args]
-NonbondedFxn = Callable[[*nonbonded_args], Energy]
```

Comment:
This syntax looks a little strange to me (should it be `Tuple[*nonbonded_args]`), and isn't accepted by my IDE:

> Unpack operator in subscript requires Python 3.11 or newer

so I opted for a more verbose version for now.